diff --git a/dwm/config.h b/dwm/config.h
index 50f8f99..44f66fc 100644
--- a/dwm/config.h
+++ b/dwm/config.h
@@ -261,8 +261,17 @@ static const char *upvol[]   = { "/usr/bin/pactl", "set-sink-volume", "0", "+5%"
 static const char *downvol[] = { "/usr/bin/pactl", "set-sink-volume", "0", "-5%",     NULL };
 static const char *mutevol[] = { "/usr/bin/pactl", "set-sink-mute",   "0", "toggle",  NULL };
 
+// https://economical-with-words.blogspot.com/2013/02/taking-screenshots-in-dwm.html
+// static const char *scrotcmd[]  = { "scrot", "--exec", "mv $f ~/Pictures/", NULL };
+// static const char *scrotfocusedcmd[]  = { "scrot", "--exec", "mv $f ~/Pictures/", "--focused", NULL };
+static const char *scrotcmd[]  = { "flameshot", "gui", NULL };
+static const char *scrotfocusedcmd[]  = { "flameshot", "screen", NULL };
+
 static const Key keys[] = {
 	/* modifier                     key            function                argument */
+  { 0,                            XK_Print,      spawn,                  {.v = scrotcmd } },
+  { ShiftMask,                    XK_Print,      spawn,                  {.v = scrotfocusedcmd } },
+  { ControlMask,                  XK_Print,      spawn,                  SHCMD("sleep 1s;scrot --select") },
 	{ 0,                            XF86XK_AudioLowerVolume,  spawn,       {.v = downvol } },
 	{ 0,                            XF86XK_AudioMute,         spawn,       {.v = mutevol } },
 	{ 0,                            XF86XK_AudioRaiseVolume,  spawn,       {.v = upvol   } },
