diff --git a/dwm/config.h b/dwm/config.h
index af451a6..971fe1a 100644
--- a/dwm/config.h
+++ b/dwm/config.h
@@ -253,10 +253,16 @@ static const char *dmenucmd[] = {
 };
 static const char *termcmd[]  = { "xfce4-terminal", NULL };
 
-
+// https://gist.github.com/palopezv/efd34059af6126ad970940bcc6a90f2e
+static const char *upvol[]   = { "/usr/bin/pactl", "set-sink-volume", "0", "+5%",     NULL };
+static const char *downvol[] = { "/usr/bin/pactl", "set-sink-volume", "0", "-5%",     NULL };
+static const char *mutevol[] = { "/usr/bin/pactl", "set-sink-mute",   "0", "toggle",  NULL };
 
 static const Key keys[] = {
 	/* modifier                     key            function                argument */
+	{ 0,                            XF86XK_AudioLowerVolume,  spawn,       {.v = downvol } },
+	{ 0,                            XF86XK_AudioMute,         spawn,       {.v = mutevol } },
+	{ 0,                            XF86XK_AudioRaiseVolume,  spawn,       {.v = upvol   } },
 	{ MODKEY,                       XK_p,          spawn,                  {.v = dmenucmd } },
 	{ MODKEY|ShiftMask,             XK_Return,     spawn,                  {.v = termcmd } },
 	{ MODKEY,                       XK_b,          togglebar,              {0} },
diff --git a/dwm/dwm.c b/dwm/dwm.c
index 3df66c0..84282ea 100644
--- a/dwm/dwm.c
+++ b/dwm/dwm.c
@@ -40,6 +40,7 @@
 #include <X11/extensions/Xinerama.h>
 #endif /* XINERAMA */
 #include <X11/Xft/Xft.h>
+#include <X11/XF86keysym.h>
 
 #include "drw.h"
 #include "util.h"
